include/nbl/asset/CCPUMeshPackerV2.h

```diff
@@ -202,12 +203,12 @@ bool CCPUMeshPackerV2<MDIStructType>::commit(IMeshPackerBase::PackedMeshBufferDa
 }
 
 template <typename MDIStructType>
-typename CCPUMeshPackerV2<MDIStructType>::PackerDataStore CCPUMeshPackerV2<MDIStructType>::createGPUPackerDataStore(video::IVideoDriver* driver)
+typename CCPUMeshPackerV2<MDIStructType>::PackerDataStore<video::IGPUBuffer> CCPUMeshPackerV2<MDIStructType>::createGPUPackerDataStore(video::IVideoDriver* driver)
 {
-    PackerDataStore m_output;
-    m_output.MDIDataBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_MDIBuffer->getSize(), m_MDIBuffer->getPointer());
-    m_output.vertexBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_vtxBuffer->getSize(), m_vtxBuffer->getPointer());
-    m_output.indexBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_idxBuffer->getSize(), m_idxBuffer->getPointer());
+    PackerDataStore<video::IGPUBuffer> m_output;
+    m_output.MDIDataBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_packerDataStore.MDIDataBuffer->getSize(), m_packerDataStore.MDIDataBuffer->getPointer());
+    m_output.vertexBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_packerDataStore.vertexBuffer->getSize(), m_packerDataStore.vertexBuffer->getPointer());
+    m_output.indexBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(m_packerDataStore.indexBuffer->getSize(), m_packerDataStore.indexBuffer->getPointer());
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/33/commits/a2e40f000ad459cd3a43fb35d94a9e5453ffcb0d#r560846913

Reply:
@Crisspl link doesn't work

Reply:
this one i believe https://github.com/Devsh-Graphics-Programming/Nabla/pull/33#discussion_r560846913